docs/how-to-guides/curating-the-editor-experience.md

```diff
@@ -413,6 +413,14 @@ To opt into this, include `core/post-content` in the Block Types for your patter
 
 Read more about this functionality in the [Page creation patterns in WordPress 6.0 dev note](https://make.wordpress.org/core/2022/05/03/page-creation-patterns-in-wordpress-6-0/) and [note that WordPress 6.1 brought this functionality to all post types](https://make.wordpress.org/core/2022/10/10/miscellaneous-editor-changes-for-wordpress-6-1/#start-content-patterns-for-all-post-types).  
 
+**Prioritize starter patterns for template creation**
+
+In the same way patterns can be prioritized for new posts or pages, the same experience can be added to the template creation process. When patterns declare support for the 'templateTypes' property, the patterns will appear anytime a template that matches the designation are created along with the options to start from a blank state or use the current fallback of the template. By default, WordPress does not include any of these patterns. 
+
+To opt into this, a pattern needs to specify a property called `templateTypes`, which is an array containing the templates where the patterns can be used as the full content. [Here's an example](https://gist.github.com/annezazu/99ce2f84fa033dae48eacc50a8287bae) of a pattern that would appear when creating a 404 template.
```

Comment:
I still always find it confusing whether to reference the properties in the form for manually registering patterns via the `register_block_pattern` function or the way it should be added to the comment header in file inside the `patterns/` directory of a theme... `Template Types` or `templateTypes` =D

Reply:
Great call out. I think that should be addressed elsewhere though, like the Theme Handbook, and we can link off. Going to leave for now but keep it in mind :)